src/sentry/static/sentry/app/components/onboardingWizard/confirmation.jsx

```diff
@@ -14,18 +22,39 @@ class Confirmation extends React.Component {
   };
 
   render() {
+    const {dismiss, hide, organization} = this.props;
+
     return (
-      <div className="ob-confirmation" onClick={this.props.dismiss}>
-        <h3>{t('Need help?')}</h3>
-        <p>
-          <a href="mailto:support@sentry.io?subject=Help with onboarding">
-            {t('Ask us!')}
-          </a>{' '}
+      <Container hide={hide} onClick={dismiss}>
+        <Header>{t('Want help?')}</Header>
+        <div>
+          <Button priority="link" to={`/settings/${organization.slug}/support/`}>
+            {t('Go to Support')}
+          </Button>{' '}
           · <a onClick={this.skip}>{t('Skip')}</a>
-        </p>
-      </div>
+        </div>
+      </Container>
     );
   }
 }
 
-export default Confirmation;
+const Container = styled('div')`
+  position: absolute;
+  top: 0px;
+  display: ${p => {
+    return p.hide ? 'none' : 'flex';
+  }};
+  bottom: 0px;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  background: rgba(255, 255, 255, 0.65);
+  width: 100%;
+  left: 0px;
```

Comment:
nit: can you move the `top`, `left`, bottom` attributes all close to each other?